src/Quest.sol

```diff
@@ -139,9 +145,9 @@ contract Quest is ERC721 {
         address _to,
         uint256 _id
     ) public override onlyAdmin {
-        require(_from == _ownerOf[_id], "WRONG_FROM");
+        if(_from == _ownerOf[_id]) revert InvalidFrom();
```

Comment:
Since you're switching from a `require` to an `if` your logic needs to be reversed.

`_from != _ownerOf[_id]`